.github/workflows/ci-code.yml

```diff
@@ -122,7 +128,10 @@ jobs:
       uses: ./.github/actions/install-aiida-core
       with:
         python-version: '3.12'
-        from-lock: 'false'
+        from-lock: 'true'
+        # NOTE: The `verdi devel check-undesired-imports` fails if
+        # the 'tui' extra is installed.
```

Comment:
This appears now because with the changes in the `install-aiida-core` we install all extras? Isn't this something we should fix later on?

Reply:
Ah sorry for being not clear. I am fine with the changes, just would like to know if we should make an issue and it fix in some later PR

Reply:
I think this is minor enough that it could go in here, but let me know if you disagree.
I can open an issue about converting more CI jobs to use the lockfile though.

Reply:
I mean that `verdi devel check-undesired-imports` fails when tui is installed sounds undesirable and sounds like this should be fixed like `only check for asyncio if tui is not installed`. I am not sure why this happens

Reply:
Aaah, now I see what you mean. Sorry, I misunderstood your original comment. 
Yeah, I'll open an issue for that.

Reply:
Opened #6691 to follow up on this